tests/NuGet.IndexingTests/ResponseFormatterTests.cs

```diff
@@ -65,16 +65,54 @@ public void WriteV2CountResultTest(int totalHits, string expected)
             Dictionary<string, string> commitUserData,
             string expected)
         {
-            var searcher = new MockSearcher(indexName, numDocs, commitUserData);
+            var currentTime = DateTime.UtcNow;
+            var searcher = new MockSearcher(indexName, numDocs, commitUserData, versions: null, reloadTime: currentTime, lastModifiedTimeForAuxFiles: null, machineName: "TestMachineX");
```

Comment:
You are right, I will merge the two tests together, however, I think we should keep the member data separate as its testing two different sets of data.